packages/turf-line-offset/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@turf/line-offset",
+  "version": "4.1.0",
+  "description": "turf line offset module",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "files": [
+    "index.js",
+    "index.d.ts"
+  ],
+  "scripts": {
+    "test": "node test.js",
+    "bench": "node bench.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Turfjs/turf.git"
+  },
+  "keywords": [
+    "line",
+    "linestring",
+    "turf",
+    "offset"
+  ],
+  "author": "Turf Authors",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Turfjs/turf/issues"
+  },
+  "homepage": "https://github.com/Turfjs/turf",
+  "devDependencies": {
+    "benchmark": "^1.0.0",
+    "tape": "^3.5.0",
+    "@turf/helpers": "^4.1.0"
+  },
+  "dependencies": {
+    "@turf/helpers": "^4.1.0",
+    "@turf/invariant": "^4.2.0",
+    "@turf/meta": "^4.2.0",
+    "intersection": "0.0.1"
```

Comment:
Oooohh, yes that's completely different, I wonder if it's worth including that logic directly in Turf. I'm not a big fan of external dependencies, especially if it can be refactored in less than 30-50 LOC